src/cockpit/channel.py

```diff
@@ -71,7 +71,7 @@ def do_control(self, command, message):
         elif command == 'options':
             self.do_options(message)
 
-    def do_channel_control(self, command, message):
+    def do_channel_control(self, channel, command, message):
```

Comment:
Hmm, shouldn't some flake/lint/whatever complain about the unused `channel` variable here and below? I.e. these would normally be named `_channel`.

Reply:
I think it doesn't complain because this is an implementation of a virtual method, and depending on the method, the individual variable names could be considered part of the API (ie: if you wanted to use kwargs with them).

Reply:
Makes sense. Python is becoming complex.. :grin: